### PR TITLE
Bug 1882085: Pin 4.6 OVN to 20.06.2-11.el8fdp

### PR DESCRIPTION
**Pull request by alexanderConstantinescu**
This pins the 4.6 version to the version 20.06.2-11.el8fdp which is shipped with 4.6.0-0.ci-2020-09-23-155325 (latest CI release)

/assign @dcbw @trozet 

**- What this PR does and why is it needed**

**- Special notes for reviewers**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Comment by alexanderConstantinescu** [comment]
/retitle Bug 1882085: Pin 4.6 OVN to 20.06.2-11.el8fdp

File: Dockerfile [hunk 1]
```diff
@@ -40,7 +40,10 @@ RUN INSTALL_PKGS=" \
 	" && \
 	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False $INSTALL_PKGS && \
 	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False "openvswitch2.13 >= 2.13.0-52" openvswitch2.13-devel && \
-	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False "ovn2.13 >= 20.06.2-3" ovn2.13-central ovn2.13-host ovn2.13-vtep && \
+	#======== 4.6.0-0.ci-2020-09-23-155325 Version ========
+	# ovn2.13-20.06.2-11.el8fdp.x86_64
+	# Pin to that
+	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False "ovn2.13 == 20.06.2-11.el8fdp" "ovn2.13-central == 20.06.2-11.el8fdp" "ovn2.13-host == 20.06.2-11.el8fdp" "ovn2.13-vtep == 20.06.2-11.el8fdp" && \
```
Comment (dcbw) [inline Dockerfile:46]: Just pin the 'ovn2.13' package; no need to pin the other OVN ones; the RPM ensures they are all the same version.

Comment (dcbw) [reply Dockerfile:46]: eg

`"ovn2.13 = 20.06.2-11.el8fdp" ovn2.13-central ovn2.13-host ovn2.13-vtep`

Comment (alexanderConstantinescu) [reply Dockerfile:46]: Yeah, I tried something like that last time but it didn't work: https://github.com/openshift/ovn-kubernetes/pull/248#issuecomment-685794872 but I was using `==` (https://github.com/openshift/ovn-kubernetes/commit/6e6e6ad21c4a8109246c4bee5f409944deaa7fa9) so maybe that's why?

**Comment by trozet** [comment]
gcp failed one test on last run:
[sig-storage] vcp at scale [Feature:vsphere]  vsphere scale tests [Suite:openshift/conformance/parallel] [Suite:k8s]

Not sure why vsphere scale tests are attempting to run here.

/override ci/prow/e2e-gcp-ovn

**Pull request merged**